server/views/components/key-details-bar/template.njk

```diff
@@ -24,11 +24,12 @@
             <div class="pac-key-details-bar-flex">
                 <div class="pac-key-details-bar-flex__child pac-probation-status">
                     <span class="govuk-body"><strong>Probation status: </strong>
-                        {% if params.probationStatus | string | lower !== "possible ndelius record" if params.probationStatus %}
-                            {{ params.probationStatus | string | capitalize }}{% if params.breach %} (Breach){% endif %}
-                        {% else %}
-                            Possible NDelius record
-                        {% endif %}
+                         {% if params.probationStatus | string | lower === "possible ndelius record" or params.probationStatus | string | lower === "no record" if params.probationStatus %}
+                            {{ params.probationStatus | string | capitalize }}
+                         {% else %}
+                             {{ params.probationStatus | string | capitalize + ' (PSR)' }}
```

Comment:
I believe that this is dependent on the new `awaitingPsr` flag in the data, not to be displayed all the time.